client/src/app/controller.coffee

```diff
@@ -0,0 +1,21 @@
+$ = require('jquery')
+Marionette = require('backbone.marionette')
+ListSitesView = require('./views/list-sites')
+FormView = require('./views/form')
+Layout = require('./views/layout')
+layout = new Layout()
+layout.render()
+
+showInRoot = (view) ->
+  layout.root.show(view.render())
```

Comment:
dont think we need to call `render()` here explicitly.  Marionetts `show` just takes the view to show and handles calling `render` for you